cmd/version.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Version Contains the version number for a particular Resumic build
+	version = "dev"
+	// GitCommit contains the commit that this build is based on
+	gitCommit = "none"
```

Comment:
Thinking about it I think `unknown` is factually more correct. Better not listen all the time ;)

Reply:
Changed!